Description: drop unneeded cleanup plugin
Author: Yadd <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2021-11-26

--- a/rollup.config.js
+++ b/rollup.config.js
@@ -1,4 +1,3 @@
-import cleanup from 'rollup-plugin-cleanup'
 import ts from 'rollup-plugin-ts'
 import transformDefaultExport from 'ts-transform-default-export'
 
@@ -18,10 +17,6 @@
       transformers: ({ program }) => ({
         afterDeclarations: transformDefaultExport(program)
       })
-    }),
-    cleanup({
-      comments: 'none',
-      extensions: ['*']
     })
   ]
 }
